includes/admin/customers/customer-actions.php

```diff
@@ -447,15 +447,15 @@ function edd_customer_delete( $args = array() ) {
 	if ( $customer->id > 0 ) {
 
 		$payments_array = explode( ',', $customer->payment_ids );
-		$success        = edd_delete_customer( $customer->id );
+		$success        = edd_destroy_customer( $customer->id );
 
 		if ( $success ) {
 
 			if ( $remove_data ) {
 
 				// Remove all payments, logs, etc
 				foreach ( $payments_array as $payment_id ) {
-					edd_delete_purchase( $payment_id, false, true );
+					edd_destroy_order( $payment_id, false, true );
```

Comment:
```suggestion
					edd_destroy_order( $payment_id );
```
This function only takes one parameter.